package.json

```diff
@@ -9,54 +9,39 @@
-    "@agoric/ertp": "0.16.3-u14.0",
-    "@agoric/governance": "0.10.4-u14.0",
-    "@agoric/store": "0.9.3-u14.0",
-    "@agoric/xsnap": "0.14.3-u14.0",
-    "@agoric/vat-data": "0.5.3-u14.0",
-    "@endo/bundle-source": "2.5.2-upstream-rollup",
-    "@endo/captp": "3.1.1",
-    "@endo/compartment-mapper": "0.8.4",
-    "@endo/far": "0.2.18",
-    "@endo/init": "0.5.56",
-    "@endo/marshal": "0.8.5",
-    "@endo/nat": "4.1.27",
-    "@endo/promise-kit": "0.2.56",
-    "@endo/lockdown": "0.1.28",
-    "@endo/import-bundle": "0.3.4",
-    "@endo/base64": "0.2.31",
-    "@endo/zip": "0.2.31",
-    "@endo/eventual-send": "0.17.2",
-    "@endo/patterns": "0.2.2",
-    "@endo/stream": "0.3.25",
-    "@endo/exo": "0.2.2",
-    "@endo/pass-style": "0.1.3",
-    "@endo/check-bundle": "0.2.18",
-    "@endo/ses-ava": "0.2.40",
-    "@endo/netstring": "0.3.26",
-    "@endo/stream-node": "0.2.26",
-    "@agoric/sharing-service": "^0.2.12-u14.0"
-  },
+  "packageManager": "yarn@4.3.1",
   "scripts": {
-    "postinstall": "patch-package",
     "start:docker": "cd contract && docker compose up -d",
     "stop:docker": "cd contract && docker compose down",
     "docker:logs": "cd contract; docker compose logs --tail 200 -f",
     "docker:bash": "cd contract; docker compose exec agd bash",
     "docker:make": "cd contract; docker compose exec agd make -C /workspace/contract",
     "make:help": "make -C contract list",
     "start:contract": "cd contract && yarn start",
+    "start:contract-build": "cd contract && yarn build",
     "start:ui": "cd ui && yarn dev",
     "lint": "yarn workspaces run lint",
     "lint:fix": "yarn workspaces run lint:fix",
     "test": "yarn workspaces run test",
     "test:e2e": "yarn workspace agoric-basics-ui test:e2e",
-    "build": "yarn workspaces run build"
+    "build": "yarn workspaces run build",
+    "cpfiles": "cd contract; make copy-bundles",
+    "rmfiles": "cd contract; make rm-bundles",
+    "provision": "cd contract; make provision-smart-wallet",
+    "fund": "cd contract; make fund",
+    "contract:deploy": "cd contract; make dev-working"
   },
-  "dependencies": {
-    "patch-package": "^8.0.0"
+  "devDependencies": {
+    "typescript": "^5.5.2"
+  },
+  "resolutions": {
+    "bn.js@npm:^5.2.0": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^4.11.9": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^5.2.1": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^4.11.8": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^4.1.0": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^4.0.0": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
+    "bn.js@npm:^5.0.0": "patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch",
```

Comment:
When doing `yarn` from the host machine(not sure if I should have done it from the container), this is the error message I get. I'm not familiar with how patching works but I get the feeling that maybe locally generated patch files was not pushed to github from your machine. Please let me know if I need to do anything different to get this PR up and running in my machine. @Jovonni 

```
anil@Anils-MacBook-Pro dapp-orchestration-basics % yarn                           
➤ YN0000: · Yarn 4.3.1
➤ YN0000: ┌ Project validation
➤ YN0057: │ dapp-agoric-orca-contract: Resolutions field will be ignored
➤ YN0000: └ Completed
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: bn.js@patch:bn.js@npm%3A4.12.0#~/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch: ENOENT: no such file or directory, open '/Users/anil/WebstormProjects/agoric-samples/dapp-orchestration-basics/.yarn/patches/bn.js-npm-4.12.0-3ec6c884f6.patch'
➤ YN0000: └ Completed in 0s 266ms
➤ YN0000: · Failed with errors in 0s 273ms
```

Reply:
@Jovonni let's find a way to drop these resolutions. As of https://github.com/Agoric/agoric-sdk/pull/9691 there's no need to patch bn.js.

The change hasn't made it to a number release so you'll have to use dev build of `@agoric/cosmic-proto@dev`. I see your package.json does that already. If it's not working you may have to just bump which dev build is in the yarn.lock

Reply:
I just pushed the patch files @turadg , should fix it for now:
https://github.com/Agoric/dapp-orchestration-basics/tree/dev/orca-v1/.yarn/patches

also, I can now try it without the patch files 🚀